migrations/2025-10-15-114811-0000_merge-modlog-tables/up.sql

```diff
@@ -0,0 +1,260 @@
+-- New enum with all possible mod actions
+-- TODO: We could also remove the Admin/Mod prefix
+CREATE TYPE modlog_kind AS enum (
+    'AdminAdd',
+    'AdminBan',
+    'AdminAllowInstance',
+    'AdminBlockInstance',
+    'AdminPurgeComment',
+    'AdminPurgeCommunity',
+    'AdminPurgePerson',
+    'AdminPurgePost',
+    'ModAddToCommunity',
+    'ModBanFromCommunity',
+    'ModFeaturePostCommunity',
+    'AdminFeaturePostSite',
+    'ModChangeCommunityVisibility',
+    'ModLockPost',
+    'ModRemoveComment',
+    'AdminRemoveCommunity',
+    'ModRemovePost',
+    'ModTransferCommunity',
+    'ModLockComment'
```

Comment:
I think its good to keep the admin / mod distinction, if only as a sanity check for all the API actions. 

We're removing a lot of strict typing by doing this (even if the database will throw an error, the struct is not going to be so clean).